dj_debexpo/dj_debexpo/wsgi.py

```diff
@@ -0,0 +1,14 @@
+"""
+WSGI config for dj_debexpo project.
+
+It exposes the WSGI callable as a module-level variable named ``application``.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.7/howto/deployment/wsgi/
+"""
+
+import os
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "dj_debexpo.settings")
+
+from django.core.wsgi import get_wsgi_application
+application = get_wsgi_application()
```

Comment:
Yay WSGI.

Relevant reading for figuring out how we'll do our future URL dispatch:

* http://flask.pocoo.org/docs/0.10/patterns/appdispatch/

* https://code.djangoproject.com/ticket/12091

* https://alexgaynor.net/2010/jan/11/hot-django-wsgi-action-announcing-django-wsgi/

Looks like via the last URL there, we can wrap the Turbogears code in `django_wsgi.django_view()` and place it in the urlconf as if it were a normal Django view.